ci/jenkins/jobs/macros.yaml

```diff
@@ -275,3 +275,55 @@
           set -e 
           DOCKER_REGISTRY="$(head -n1 ci/docker-registry)"
           ./ci/jenkins/test-mc.sh --testcase e2e --registry ${DOCKER_REGISTRY} --mc-gateway --codecov-token "${CODECOV_TOKEN}" --coverage --kind
+- builder:
+    name: builder-ipv6-e2e-jumper
+    builders:
+      - shell: |-
+          #!/bin/bash
+          set -e
+          DOCKER_REGISTRY="$(head -n1 ci/docker-registry)"
+          ./ci/jenkins/test.sh --testcase e2e --registry ${DOCKER_REGISTRY} --testbed-type jumper
+
+- builder:
+    name: builder-ipv6-conformance-jumper
+    builders:
+      - shell: |-
+          #!/bin/bash
+          set -e
+          DOCKER_REGISTRY="$(head -n1 ci/docker-registry)"
+          ./ci/jenkins/test.sh --testcase conformance --registry ${DOCKER_REGISTRY} --testbed-type jumper
+- builder:
+    name: builder-ipv6-networkpolicy-jumper
```

Comment:
Can we use three builder for both ds and ipv6-only jobs?

Reply:
Also please add the issue info in commit message.

Reply:
The DS jobs are different from IPv6 jobs, they need to use `sed` command to change file. So I used different jobs for DS and IPv6, maybe it's more flexible.

Reply:
Issue info added.

Reply:
> The DS jobs are different from IPv6 jobs, they need to use `sed` command to change file. So I used different jobs for DS and IPv6, maybe it's more flexible.

Could you explain why we need `sed` in DS jobs?

Reply:
I'm not sure.But in the previous Jenkins jobs, the DS jenkins job needs the `sed` operation to change variable `DOCKER_BUILD_ARGS`.

Reply:
I don't see any reason to include the `sed` command in this script. Could you check if all the ds jobs work well without this command? If they do, we can remove unnecessary job builders.

Reply:
Sure. I'll check it.

Reply:
After verification, the `sed` command can be removed. I have removed this line, both IPv6 and Dual-stack jobs use the same macroes now.